docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  mysql:
+    image: mysql:5.7
+    restart: on-failure
+    environment:
+      - MYSQL_ROOT_PASSWORD=inventory_root_password
+      - MYSQL_HOST=inventory_mysql
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=inventory_local
+      - MYSQL_USER=inventory_test_user
+      - MYSQL_PASSWORD=inventory_test_password
```

Comment:
Can you make these credentials shorter? 
`MYSQL_ROOT_PASSWORD=ci_root_pw`
`MYSQL_USER=ci_user`
`MYSQL_PASSWORD=ci_test_password`
Call me lazy but we don't want long password for local DB set up.

Reply:
Done.